Guided Hunting - Anomalous Office365 Exchange Sessions.ipynb

```diff
@@ -0,0 +1,547 @@
+{
```

Comment:
<p>Could we do some formatting of the output so that it easier to see the events/commands/parameters.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/31/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>